modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -1037,6 +1094,49 @@ struct MishFunctor : public BaseDefaultFunctor<MishFunctor>
         return (x * n) / (n + 2.f);
     }
 
+    // failed at Test_Darknet_nets.YOLOv4/0, Test_Darknet_nets.YOLOv4/1, Test_Int8_nets.YOLOv4/0
+//     void apply(const float* srcptr, float* dstptr, int stripeStart, int len, size_t planeSize, int cn0, int cn1) const {
+//         CV_UNUSED(stripeStart);
+//         for (int cn = cn0; cn < cn1; cn++, srcptr += planeSize, dstptr += planeSize) {
+//             int i = 0;
+// #if (CV_SIMD || CV_SIMD_SCALABLE)
+//             // x * (exp(x) + 2.f) * exp(x)) / (exp(x) + 2.f) * exp(x) + 2.f)
+//             v_float32 two = vx_setall_f32(2.0f);
+//             for (; i <= len - vlanes; i += vlanes) {
+//                 if (i + vlanes > len) {
+//                     if (i == 0 || i == len) {
+//                         break;
+//                     }
+//                     i = len - vlanes;
+//                 }
+//                 v_float32 x = vx_load(srcptr + i);
+
+//                 // exp(x)
+//                 v_float32 expx = v_exp(x);
```

Comment:
Got accuracy problem with `v_exp`. It goes to `nan` with the second batch for some reason. Tested with the image in the second batch, it still goes no detection (nan).